frontend/src/i18n/locales/fr/translation.json

```diff
@@ -315,6 +315,15 @@
   "Default Request": "Demande par défaut",
   "Max": "Max",
   "Min": "Min",
+  "Invalid namespace name: name must be 63 characters or shorter": "",
+  "Invalid namespace name: name must start with an alphanumeric character": "",
+  "Invalid namespace name: name must end with an alphanumeric character": "",
+  "Invalid namespace name: name must consist of lower case alphanumeric characters or -": "",
+  "Creating namespace {{  name  }}…": "",
+  "Cancelled creation of {{  name  }}.": "",
+  "Created namespace {{  name  }}.": "",
+  "Error creating namespace {{  name  }}.": "",
+  "Create Namespace": "",
```

Comment:
I wonder if in the interest of not having to translate 5 strings per resource, we can reuse the string "Failed to create {{ kind }} {{ name }}." and similar.